JapaneseFilter/sections/specific.txt

```diff
@@ -43,6 +43,8 @@ hentai-covers.site,109cinemas.net,4545.to,bunshun.jp,eikaiwa.dmm.com,gizmodo.jp,
 !
 !
 !
+jp.reuters.com##.bleeding.module
+jp.reuters.com##div[style].module-interactive ~ div.module
```

Comment:
These spaces are too small. No one notices them) Only this user often reports them(on Japanese sites).